WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationForm.swift

```diff
@@ -182,11 +188,12 @@ private extension BlazeCampaignCreationForm {
                 // Image
                 Image(uiImage: viewModel.image?.image ?? .blazeProductPlaceholder)
                     .resizable()
-                    .aspectRatio(contentMode: .fill)
+                    .aspectRatio(contentMode: .fit)
```

Comment:
Should we update the `BlazeEditAdView` image's aspect ratio as well?

Reply:
Done in 0b423c97100bdcd4eeeee1046f135f69450513c3.